### PR TITLE
Added more questions and made the skill working

### LINKED ISSUE #160: Cricket skill not working
**Actual Behaviour**
While implementing etherpad skill, I implemented back to back skill. back to back means user ask for current matches going on. this displays all the natches along with their match ids. While dreaming, user enters the match id and hence gets the results of that particular match. 

**Expected Behaviour**
Questions should be made and placed in the skill such that user has best possible experience. 

State here what the feature should enable the user to do.
Refactor the skill cricket.

**Screenshots of the issue**
screenshot 33

**Would you like to work on the issue?**
yes

### PR DESCRIPTION
**Pull request by DravitLochan**
Fixes #160 

Changes: changed the questions

### CODE REVIEW

**Comment by DravitLochan** [comment]
@dynamitechetan @saurabhjn76 @madhavrathi @chashmeetsingh @AliAyub007 @isuruAb 
Please review

File: models/general/entertainment/en/cricket.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-matches going on|current matches|score *
+matches going on|current matches|get me a list of matches going on| cricket matches
```
Comment (AliAyub007) [inline models/general/entertainment/en/cricket.txt]: You can add more questions to it I think. 
Do you know about matches going on?
Do you know cricket matches going on?
Tell me about matches going on?
Tell me about current cricket matches. 

and same variations for all.

Comment (DravitLochan) [reply models/general/entertainment/en/cricket.txt]: Okay!

Comment (isuruAb) [reply models/general/entertainment/en/cricket.txt]: is there any cricket match today?
what are the cricket matches going on ?

**Review by isuruAb — COMMENTED** [review]
add more questions 👍

**Comment by hpdang** [comment]
@DravitLochan please take care of your reviewers' comments

**Comment by DravitLochan** [comment]
Updating the pr!

**Comment by DravitLochan** [comment]
@isuruAb @AliAyub007 I have updated the pr. Please review!

**Review by madhavrathi — APPROVED** [review]

**Review by dynamitechetan — APPROVED** [review]

**Review by isuruAb — APPROVED** [review]
LGTM 👍

**Pull request merged**